packages/math/src/Point.ts

```diff
@@ -1,13 +1,29 @@
-import { IPoint } from './ObservablePoint';
+/**
+ * Common interface for points. Both Point and ObservablePoint implement it
+ * @memberof PIXI
+ * @interface IPoint
+ * @property {number} x - x coord
+ * @property {number} y - y coord
```

Comment:
@ivanpopelyshev if you move this below the initialisation then the jsdoc comment is not removed and the type appears in `pixi.js.d.ts` 

```js
export interface IPoint
{
    x: number;
    y: number;
    copyFrom(p: IPoint): this;
    copyTo(p: IPoint): IPoint;
    equals(p: IPoint): boolean;
    set(x?: number, y?: number): void;
}
/**
 * Common interface for points. Both Point and ObservablePoint implement it
 * @memberof PIXI
 * @interface IPoint
 * @property {number} x - x coord
 * @property {number} y - y coord
 */
```